README.md

```diff
@@ -5,6 +5,8 @@
 https://kaylemba.github.io/MyPortfolio2/
 
 This is a simple web develoment project for microverse created using simple HTML(Hyper text mark-up Language.) and CSS(Cascading Style Sheet) which is used for styling or decorating the webpage.
+# MOBILE VERSION
+![screenshot](./images/mobile-js.png) 
```

Comment:
- [ ] OPTIONAL
This image link is not working. Kindly endeavor to update the same

Reply:
done